app/pipedrive/_schema.py

```diff
@@ -136,26 +139,33 @@ async def company_dict(self) -> dict:
 class Person(PipedriveBaseModel):
     id: Optional[int] = Field(None, exclude=True)
     name: str
-    primary_email: Optional[str] = ''
+    email: Optional[str] = ''
     phone: Optional[str] = ''
-    address_country: Optional[str] = None
     owner_id: Optional[fk_field(Admin, 'pd_owner_id')] = None
     org_id: Optional[fk_field(Company, 'pd_org_id', null_if_invalid=True)] = None
 
+    # These are all custom fields
+    address_country: Optional[str] = Field(None, custom=True)
```

Comment:
Why is this a custom field? Surely it should follow the same logic as Orgs?

Reply:
Unfortunately not, there is no address field on a PD Person

Reply:
Remove then